Collect automations and ingestion_server to Sphinx docs (#2330)
WordPress/openverse: documentation/automations/index.md

# Automations

> This section contains automations for managing the Openverse repositories and
> Developer Experience.

Reviewer: Just a small change for clarity.

```suggestion
This section documents automation scripts that manage aspects of the Openverse repositories and
Developer Experience.
```
